sdk/core/src/request.rs

```diff
@@ -32,6 +35,47 @@ pub struct Request {
     body: Body,
 }
 
+impl Request {
+    fn new(uri: Uri, method: Method, headers: HeaderMap, body: Body) -> Self {
+        Self {
+            uri,
+            method,
+            headers,
+            body,
+        }
+    }
+}
+
+impl Serialize for Request {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        let mut hm = std::collections::HashMap::new();
+        for (h, v) in self.headers().iter() {
+            if h.as_str().to_lowercase() == "authorization" {
+                hm.insert(h.to_string(), "stripped");
```

Comment:
nit: maybe something like `"<<STRIPPED>>"` would be clearer.

Reply:
That works. I think most of our SDKs just use "Sanitized". In fact, from a policheck standpoint, that's probably better than "stripped". 😉